packages/optimizers/terser/src/TerserOptimizer.js

```diff
@@ -60,17 +61,24 @@ export default new Optimizer({
                 column: gen_col,
               },
             });
+
+            if (mappingsBuffer.length > 25) {
```

Comment:
Where did this length come from?

Reply:
Just random, wanted to prevent overhead for sending data between c++ and JS but I'm hoping I can just send the vlq version as it's an insane amount faster to process than js objects

Reply:
Ah ok. How would you get the vlq version from terser?

Reply:
They give the stringified version, gonna see if they also provide the object version in any way as parsing json is even slower...

Reply:
But wouldn't that mean Terser would need to encode the VLQ? Probably using the mozilla-sourcemap package? Seems like it would be slower, no?

Reply:
As far as I can tell they do it anyway our config.output.source_map hack broke I think. The mappingBuffer array was empty when I logged it